ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -42,6 +42,13 @@ const (
 	JobDeploymentStatusSuspended    JobDeploymentStatus = "Suspended"
 )
 
+type JobSubmissionMode string
+
+const (
+	K8sJobMode JobSubmissionMode = "K8sJobMode" // Submit job via Kubernetes Job
```

Comment:
nit: Is the "Mode" part neeeed? I think it's implied from the field name. Maybe just `K8sJob` and `HTTP` is enough?

Reply:
I prefer to add "Mode". Without it, users might find the documentation confusing, as they would have to guess whether "HTTP" refers to the submission mode or the protocol. In addition, "K8sJob" and "K8s Job" can also be confusing.

Reply:
> "HTTP" refers to the submission mode or the protocol

Users will rarely see "HTTP" by itself though. They will usually see `submissionMode: HTTP` and `submissionMode: batchv1.Job`.

Reply:
> They will usually see submissionMode: HTTP and submissionMode: batchv1.Job.

This is reasonable in the YAML files and in the KubeRay codebase, but it may not be accurate for the Ray documentation.